src/SFA.DAS.FindApprenticeshipTraining.Api.UnitTests/Models/WhenCastingGetLocationSearchResponseFromMediatorType.cs

```diff
@@ -12,10 +12,13 @@ public class WhenCastingGetLocationSearchResponseFromMediatorType
         [Test, AutoData]
         public void Then_The_Fields_Are_Correctly_Mapped(GetLocationsQueryResponse source)
         {
+            //Arrange
+            source.Locations = source.Locations.Select(c => { c.Postcode = null; return c; }).ToList();
+
             var actual = (GetLocationSearchResponse) source;
             
-            actual.Locations.ToList().First().Should().BeEquivalentTo(source.Locations.First());
-            actual.Locations.ToList().Last().Should().BeEquivalentTo(source.Locations.Last());
+            actual.Locations.ToList().First().Should().BeEquivalentTo(source.Locations.First(), options=> options.ExcludingMissingMembers());
+            actual.Locations.ToList().Last().Should().BeEquivalentTo(source.Locations.Last(), options=> options.ExcludingMissingMembers());
             actual.Locations.Count().Should().Be(source.Locations.Count());
         }
     }
```

Comment:
Could do with some tests/assertions showing that the Name is a concatenation of the LocationName and LocalAuthorityName when the postcode is null, and also for using the postcode when that is not null or empty string